CBLClient/Apps/CBLTestServer-Java-Desktop/src/main/java/com/couchbase/mobiletestkit/javatestserver/TestServerLogger.java

```diff
@@ -0,0 +1,35 @@
+package com.couchbase.mobiletestkit.javatestserver;
+
+import com.couchbase.mobiletestkit.javacommon.util.Logger;
+
+public class TestServerLogger implements Logger {
+    @Override
+    public void i(String tag, String msg) {
+        System.out.println("INFO " + tag + ": " + msg);
+    }
+
+    @Override
+    public void d(String tag, String msg) {
+        System.out.println("DEBUG " + tag + ": " + msg);
+    }
+
+    @Override
+    public void e(String tag, String msg) {
+        System.out.println("ERROR " + tag + ": " + msg);
+    }
+
+    @Override
+    public void e(String tag, String msg, Throwable throwable) {
+        System.out.println("ERROR " + tag + ": " + msg + " - " + throwable);
+    }
+
+    @Override
+    public void w(String tag, String msg) {
+        System.out.println("WARNING " + tag + ": " + msg);
+    }
+
+    @Override
+    public void w(String tag, String msg, Throwable throwable) {
```

Comment:
THis is going to print only the error message from the Throwable, not the stack trace.  You should `printStackTrace`

Reply:
fixed